source/_integrations/snmp.markdown

```diff
@@ -358,6 +358,10 @@ command_payload_off:
   description: The value to write to turn off the switch, if different from `payload_off`.
   required: false
   type: string
+unique_id:
+  description: An ID that uniquely identifies this entity. This allows changing the `name`, `icon` and `entity_id` from the web interface.
```

Comment:
Name, icon & entity_id are not code... they are UI options. So let's not code-fence them.,

```suggestion
  description: An ID that uniquely identifies this entity. This allows changing the name, icon, and entity ID from the user interface.
```